services/QuillLMS/app/services/stripe_integration/webhooks/checkout_session_completed_event_handler.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  module Webhooks
+    class CheckoutSessionCompletedEventHandler < EventHandler
+      def run
+        stripe_checkout_session&.destroy!
```

Comment:
Is there any value in saving this record and doing a safe delete (e.g. setting a `finished_at`/`expired_at` or a `processed/expired` state)? For example, would it be helpful for debugging checkout issues or giving a full timeline/understanding of the checkout process? It may not be that helpful, so we can delete, but it feels worth asking that question.

Reply:
I've kind of been going back and forth on [this](https://github.com/empirical-org/Empirical-Core/pull/9247#discussion_r892589050), but I think deleting them is the most practical approach.  We can recover some of the information via `checkout.session.expired`, but if we end up needing more, I might add the `expiration` back in.